src/tests/functional/plugin/shared/src/behavior/plugin/life_time.cpp

```diff
@@ -77,14 +75,22 @@ namespace BehaviorTestsDefinitions {
             release(i);
     }
 
-    TEST_P(HoldersTest, Orders) {
+    TEST_P(HoldersDeathTest, Orders) {
         // Test failed if crash happens
+#if defined(_WIN32)
+        EXPECT_NO_THROW(release_order_test(order, targetDevice, function));
+#else
         EXPECT_NO_CRASH(release_order_test(order, targetDevice, function));
+#endif
     }
 
-    TEST_P(HoldersTestImportNetwork, Orders) {
+    TEST_P(HoldersDeathTestImportNetwork, Orders) {
         // Test failed if crash happens
+#if defined(_WIN32)
+        EXPECT_NO_THROW(release_order_test(order, targetDevice, function));
+#else
         EXPECT_NO_CRASH(release_order_test(order, targetDevice, function));
```

Comment:
Maybe I missed something, but on my Ubuntu20 this EXPECT_NO_CRASH macro didn't work. With both "fast" & "threadsafe" flags. To be honest I didn't test it on windows...

Reply:
With "fast" it worked for me in debug mode and looks like in ci too, but if it found, that there are some other threads it will fail, so, yes, fast is not appropriate here